picky-krb/src/crypto/mod.rs

```diff
@@ -35,6 +35,12 @@ pub enum KerberosCryptoError {
     CipherPad(PadError),
     #[error("Invalid seed bit len: {0}")]
     SeedBitLen(String),
+    #[error(transparent)]
+    RandError(#[from] rand::rand_core::OsError),
+    #[error(transparent)]
+    TooSmallBuffer(#[from] inout::OutIsTooSmallError),
+    #[error(transparent)]
+    ArrayTryFromSliceError(#[from] std::array::TryFromSliceError),
```

Comment:
issue: This error is quite opaque. It would be better to use a different name matching the context.

Reply:
Removed the error